data/player/Admin.json

```diff
@@ -1 +1 @@
-{"attributes":{"maxHealth":100,"health":100,"maxMana":200,"mana":200,"maxEnergy":200,"energy":200,"strength":20,"intelligence":20,"wisdom":20,"dexterity":20,"constitution":20},"experience":0,"inventory":[],"level":1,"name":"Admin","prefs":[],"questData":{"completed":[],"active":[["limbo:1",{"state":{"count":0}}]]},"room":"limbo:1","equipment":{}}
\ No newline at end of file
+{"attributes":{"health":100,"mana":200,"energy":200,"strength":20,"intelligence":20,"wisdom":20,"dexterity":20,"constitution":20},"experience":0,"inventory":[],"level":1,"name":"Admin","prefs":[],"questData":{"completed":[],"active":[["limbo:1",{"state":{"count":0}}]]},"room":"limbo:1","equipment":{}}
```

Comment:
Is this correct? the serialized attributes here don't look like the format you described